bazel/python_utils/conftest.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+
+@pytest.hookimpl()
+def pytest_sessionstart(session):
+    if os.geteuid() != 0:
+        raise Exception(
+            "\n\n" + \
+            "################################################################################\n" * 3 + \
+            "To execute tests tagged as 'manual' you need to use the relevant shell script in\n" + \
+            "'$MAGMA_ROOT/bazel/scripts/'!\n" + \
+            "For Python AGW network tests: '$MAGMA_ROOT/bazel/scripts/run_sudo_tests.sh'!\n" + \
+            "For AGW integration tests: '$MAGMA_ROOT/bazel/scripts/run_integ_tests.sh'!\n" + \
+            "For AGW load tests: '$MAGMA_ROOT/bazel/scripts/run_load_tests.sh'!\n" + \
+            "The scripts can be used to execute individual tests, to have the full overview\n" + \
+            "of the available options add '--help' to them.\n" + \
```

Comment:
`to the commands above.`

I guess technically this is not true because of the load tests?

Reply:
Yes, but I do not really see this as a problem. We could also just add a little help to the load test script.

Reply:
In principle that would be good, but there is really only one option. I would be okay either way, if you want add a little help that just states the only option or we accept that it will fail sometimes, it is probably rather rare.